joomla/joomla-cms
# [5.1] Convert mod_wrapper to service provider

### Summary of Changes

Converts the wrapper module to service provider.

### Testing Instructions

1. Make sure you have site module mod_wrapper published and configured
2. You should see module output on frontend

![mod_wrapper](https://github.com/joomla/joomla-cms/assets/59166858/a824cc32-2e33-4f95-9917-14f245dabb2d)

### Actual result BEFORE applying this Pull Request

Module works as expected

### Expected result AFTER applying this Pull Request

Module works same way as before applying pull request

### Link to documentations
Please select:
